### ISSUE TITLE
ObservableProperties: unspecified operations, missing vocabulary terms

### ISSUE THREAD

**Issue by mlagally** [w3c/wot-thing-description#1404]
The obervable properties are conceptually very similar to event affordances.

A consumer can subscribe/unsubscribe to property changes. When they occur, messages are trabeing sennt to subscriber(s).

Currently the operations observeproperty and unobserveproperty are unspecified.
The operations should work in the same way as the event subsription mechanism, i.e. it should be closely aligned with the EventAffordance.

Perhaps it should be considered to make properties inherit the event affordance?

**Comment by egekorkan** [comment]
I think that would complicate property affordance quite a lot for a bit of an edge use case. I think that if such observation mechanisms are needed, one should use event affordance.

**Comment by mlagally** [comment]
@egekorkan 
I agree that the observe mechanism would be more complicated - unfortunately the current observe mechanism seems to be underspecified.
